src/main/java/org/opentripplanner/analyst/Histogram.java

```diff
@@ -16,80 +19,130 @@
 public class Histogram implements Serializable {
 
     /**
+     * The weighting functions to be used, as an array. Generally this will be an array of 120
+     * functions, one to calculate cumulative accessibility for each minute in two hours.
+     * But any additive function can be used, and the output of the functions will be places in
+     * counts and sums parallel to this array.
+     */
+    public static WeightingFunction[] weightingFunctions;
+
+    /**
+     * The steepness of the logistic rolloff, basically a smoothing parameter.
+     * Must be negative or your results will be backwards (i.e. jobs nearby will be worth less than jobs far away).
+     * 
+     * The larger it is in magnitude, the less smoothing. setting it to -2 / 60.0 yields a rolloff of about 5 minutes.
+     */
+    // TODO: should not be final, but that means that we need to rebuild the weighting functions when it is changed. 
+    public static final double LOGISTIC_STEEPNESS = -2 / 60.0;
+
+    static {
+        weightingFunctions = new WeightingFunction[120];
+
+        for (int i = 0; i < 120; i++) {
```

Comment:
At some point we should take a stand on how many minutes of travel time we support. There have always been a few places throughout the code where we assume 1.5 or 2 hours. We should pick one and make it "official". 120 minutes is very practical because it fits in both positive and negative form into a single byte. I'm not sure whether we'll use that fact, but it's super convenient when using greyscale rasters and byte arrays to carry data.